tests/queries/0_stateless/02477_age.sql

```diff
@@ -0,0 +1,82 @@
+SELECT 'Various intervals';
+
+SELECT age('year', toDate('2017-12-31'), toDate('2016-01-01'));
+SELECT age('year', toDate('2017-12-31'), toDate('2017-01-01'));
+SELECT age('year', toDate('2017-12-31'), toDate('2018-01-01'));
+SELECT age('quarter', toDate('2017-12-31'), toDate('2016-01-01'));
+SELECT age('quarter', toDate('2017-12-31'), toDate('2017-01-01'));
+SELECT age('quarter', toDate('2017-12-31'), toDate('2018-01-01'));
+SELECT age('month', toDate('2017-12-31'), toDate('2016-01-01'));
+SELECT age('month', toDate('2017-12-31'), toDate('2017-01-01'));
+SELECT age('month', toDate('2017-12-31'), toDate('2018-01-01'));
+SELECT age('week', toDate('2017-12-31'), toDate('2016-01-01'));
+SELECT age('week', toDate('2017-12-31'), toDate('2017-01-01'));
+SELECT age('week', toDate('2017-12-31'), toDate('2018-01-01'));
```

Comment:
I am confused. On my system and in the expected results,

```sql
SELECT age('month', toDate('2017-12-31'), toDate('2018-01-01'));
-- 0
```

but

```sql
SELECT age('week', toDate('2017-12-31'), toDate('2018-01-01'));
-- 1
```

Is that because of funny time zone effects?

Reply:
I am confused too. Because I get 0 when I run the query form clickhouse-client and 1 then the query is run from test. I am working on it.

Reply:
Found the issue and corrected it. I will push my changes later on today.